addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -1,16 +1,30 @@
 import { defineHrModels } from "@hr/../tests/hr_test_helpers";
 import { start } from "@mail/../tests/mail_test_helpers";
-import { describe, expect, test } from "@odoo/hoot";
+import { describe, expect, test,beforeEach } from "@odoo/hoot";
 import { waitFor } from "@odoo/hoot-dom";
 import { contains, makeMockServer, mountView, onRpc } from "@web/../tests/web_test_helpers";
 import { getOrigin } from "@web/core/utils/urls";
 
 describe.current.tags("desktop");
 defineHrModels();
-
+let env;
+beforeEach(async () => {
+    const server = await makeMockServer();
+    env = server.env;
+    onRpc("hr.employee.public", "get_avatar_card_data", (params) => {
+        const resourceIdArray = params.args[0];
+        const resourceId = resourceIdArray[0];
+        const resources = env['hr.employee.public'].search_read([['id', '=', resourceId]]);
+        const result = resources.map(resource => ({
+            name: resource.name,
+            work_email:resource.work_email,
+            phone: resource.phone,
+            user_id: resource.user_id,
+        }));
+        return result;
+    })
```

Comment:
cannot we do something like this:

```suggestion
    onRpc("hr.employee.public", "get_avatar_card_data", function (params) {
        const resourceIdArray = params.args[0];
        const resourceId = resourceIdArray[0];
        const resources = this.env['hr.employee.public'].search_read([['id', '=', resourceId]]);
        const result = resources.map(resource => ({
            name: resource.name,
            work_email:resource.work_email,
            phone: resource.phone,
            user_id: resource.user_id,
        }));
        return result;
    })
```

Reply:
hello @xavierbol 

I tried to reduce the code by moving the RPC mock to beforeEach, since in the prior version it was being called in each test.

However, we shouldn't change that for now, as these tests have already been refactored in later versions using the makeMockServer approach. So I’m removing this diff from the test case.

Hope everything is clear now :smile: 
Thanks again for the review!